PowerFGT/Public/cmdb/policy.ps1

```diff
@@ -0,0 +1,303 @@
+﻿#
+# Copyright 2019, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
+# Copyright 2019, Benjamin Perrier <ben dot perrier at outlook dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+function Get-FGTPolicy {
+    
+  <#
+      .SYNOPSIS
+      Get list of all policies
+
+      .DESCRIPTION
+      Get list of all policies (name, source, destination...)
+
+      .EXAMPLE
+      Get-FGTPolicy
+
+      Get list of all policies
+
+      Get-FGTPolicy | select name, srcintf, dstintf, srcaddr, dstaddr, action, status, schedule, service, nat
+
+      Get list of all policies with a selection of attributes
+
+  #>
+
+
+        $reponse = Invoke-FGTRestMethod -uri 'api/v2/cmdb/firewall/policy' -method 'GET'
+        $reponse.results
+   }
+
+
+
+#Add-FGTPolicy -name "test_policy" -srcintf port1 -dstintf port2 -srcaddr all -dstaddr all -action "accept" -status "enable" -schedule "always" -service ALL -nat "disable"
+
+
+
+
+function Add-FGTPolicy {
+
+    <#
+        .SYNOPSIS
+        Add a FortiGate Policy
+
+        .DESCRIPTION
+        Add a FortiGate Policy
+
+        .EXAMPLE
+        Add-FGTPolicy -name "test_policy" -srcintf port1 -dstintf port2 -srcaddr all -dstaddr all -action "accept" -status "enable" -schedule "always" -service ALL -nat "disable"
+        
+        Add-FGTPolicy -name "test_policy" -srcintf port1 -dstintf port2 -srcaddr all -dstaddr all -action "accept" -status "enable" -schedule "always" -service "HTTP, HTTPS, SSH" -nat "disable"
+        MULTI SERVICE
+
+    #>
+
+
+  Param(
```

Comment:
All parameter are really Mandatory ? (specialty when $action is deny or learn...)